SU-HSDO/suhumsci
# feat(STN-469) hero opacity

# READY FOR REVIEW

## Summary
Paired with a designer to decide on new color values for Colorful theme carousel and Traditional theme carousel and heros.


- [x]  backstop testing

Browser Testing

## Steps to Test
1. `npm run test` passes
2. For colorful:
-  [ ] check the carousel page and make sure things look good.
3. For traditional:
-  [ ] check the carousel page and hero page and make sure things look good.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
